### PR TITLE
Milestone3

### PR DESCRIPTION
**Pull request by addod19**
## In this milestone, we performed the following actions:

- Make the BooksForm component stateful
    - The component should store the title and the book category in its own state
    - The component should implement the handleChange method which reacts to the user typing to input a name or to changes in the category select
    - Each change to the form should modify the component’s state
    - The component should implement handleSubmit which will save the new book in the Redux store and resets the component’s state

- Connect the REMOVE_BOOK action to the component
    - Add a new table column to the BooksList component. It should render the “Remove Book” button next to each book
    - Implement the handleRemoveBook method which accepts a book as an argument and removes it from the Redux store

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "es6": true,
     "jest": true
   },
+  "parser": "babel-eslint",
```
Comment (bellom) [inline .eslintrc.json]: Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/react-redux/.eslintrc.json)
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export const removeBook = id => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+// export const filterBook = category => ({
+//   type: CHANGE_FILTER,
+//   category,
+// });
```
Comment (bellom) [inline src/actions/index.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: src/constant/initialState.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    category: 'Sci-fi',
+  },
+  {
+    id: uuidv1(),
+    title: 'Ghost Buster',
+    category: 'Horror',
+  },
+];
+
+export default INITIAL_STATE;
```
Comment (bellom) [inline src/constant/initialState.js:31]: I will advise you to always drop a note(explain your action) once you make any changes to the previous milestone requirement.

Creating a new file for this doesn't look bad. 👍

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi Team :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by suretrust — APPROVED** [review]
Hi team, 👋 

### Great job implementing the suggested changes. 🙌 Let's get this merged! 👍 

_Good luck with your future projects and happy coding!_ 🎉

**Pull request merged**